src/main/java/core/basesyntax/strategy/operation/SupplyOperation.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+
+public class SupplyOperation implements OperationHandler {
+    private StorageDao storageDao = new StorageDaoImpl();
```

Comment:
The `storageDao` instance should not be created inside the `SupplyOperation` class. It should be passed to the constructor to ensure that the class is not dependent on a specific implementation of the `StorageDao`.